car:
  # Publish all joint states
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Back wheel effort controllers'
  back_left_wheel_effort_controller:
    type: effort_controllers/JointEffortController
    joint: back_left_wheel_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  
  back_right_wheel_effort_controller:
    type: effort_controllers/JointEffortController
    joint: back_right_wheel_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}


  front_wheels_effort_controller:
    type: effort_controllers/JointEffortController
    joint: steering_cylinder_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}